native/src/models/ContentLoadCriterion.ts

```diff
@@ -22,8 +21,8 @@ export class ContentLoadCriterion {
   }
 
   shouldUpdate(lastUpdate: Moment | null | undefined): boolean {
-    // If an update is forced or the last update was more than 24h ago it should update
-    return this._forceUpdate || !lastUpdate || lastUpdate.isBefore(moment.utc().subtract(MAX_CONTENT_AGE, 'hours'))
+    // If an update is forced or the last update was before the start of today
+    return this._forceUpdate || !lastUpdate || lastUpdate.isBefore(moment.utc().startOf('day'))
```

Comment:
Could you add a test for this criterion? :)

Reply:
@Taggotty 
added some unit tests :) let me know if that suits to requirements